src/Microsoft.Framework.PackageManager/Utils/LockFileUtils.cs

```diff
@@ -42,9 +42,17 @@ public static LockFileLibrary CreateLockFileLibrary(LockFileLibrary previousLibr
                     }
 
                     var assemblyPath = Path.Combine(installPath, filePath);
-                    if (IsAssemblyServiceable(assemblyPath))
+                    try
                     {
-                        lockFileLib.IsServiceable = true;
+                        if (IsAssemblyServiceable(assemblyPath))
```

Comment:
I don't know what caused the exception I got here but it doesn't actually seem wrong to have a guard here. We don't want a broken DLL file causing an exception during restore.